tests/ssg_test_suite/common.py

```diff
@@ -518,6 +521,28 @@ def send_scripts(test_env):
     return remote_dir
 
 
+def get_test_dir_config(test_dir, product_yaml):
+    test_config = dict()
+    test_config_filename = os.path.join(test_dir, TESTS_CONFIG_NAME)
+    if os.path.exists(test_config_filename):
+        test_config = ssg.yaml.open_and_expand(test_config_filename, product_yaml)
+    return test_config
+
+
+def select_templated_tests(test_dir_config, available_scenarios_basenames):
+    if "deny_scenarios" in test_dir_config:
+        available_scenarios_basenames = {
+            test_name for test_name in available_scenarios_basenames
+            if test_name not in test_dir_config["deny_scenarios"]
+        }
+    if "allow_scenarios" in test_dir_config:
+        available_scenarios_basenames = {
+            test_name for test_name in available_scenarios_basenames
+            if test_name in test_dir_config["allow_scenarios"]
+        }
+    return available_scenarios_basenames
```

Comment:
It means that a test scenario, which by any reason, was declared both in `allow_scenarios` and `deny_scenarios` in the same config file will be removed from the test scenarios list. In other words, `deny_scenarios` has preference over `allow_scenarios`. Although this is clear in the code, looks reasonable to have this information also clear in the documentation (README.md).

Reply:
I would rather go for "undefined" - both allow and deny accept literal strings, not regexes, and there is no config inheritance, so that request is both explicit and quite meaningless at the same time. WDYT?
An alternative would be to define it or warn/stop the user as that they probably don't want that, but I don't know whether such attention in this context makes sense.

Reply:
I'd do error and inform user. They are doing something inconsistent when they both allow and deny, that's clearly a mistake on their side. If we would default or use undefined they would wonder why some rule isn't used and they would be confused.